src_test/test_ppx_yojson.cppo.ml

```diff
@@ -1,11 +1,21 @@
 open OUnit2
 
-type json = [%import: Yojson.Safe.json] [@@deriving show]
+type json =
+  [ `Assoc of (string * json) list
+  | `Bool of bool
+  | `Float of float
+  | `Int of int
+  | `Intlit of string
+  | `List of json list
+  | `Null
+  | `String of string
+  | `Tuple of json list
+  | `Variant of string * json option ]
+  [@@deriving show]
```

Comment:
Probably it can make sense to add these functions to `yojson` itself at some point.